.travis.yml

```diff
@@ -21,27 +21,29 @@ env:
         - SETUP_CMD='test'
         - TEST_MODE='normal'
         - ATOM_DATA_URL='https://github.com/tardis-sn/tardis-atomdata/raw/master/kurucz_cd23_chianti_H_He.h5.zip'
+        - PLASMA_NLTE_REFERENCE_URL='https://github.com/vg3095/tardis-refdata/raw/plasma-ref2/plasma_reference/plasma_nlte_reference.h5'
```

Comment:
@vg3095 can you check out how this refdata is  solved in carsus  and use the same methods